README.md

```diff
@@ -725,6 +726,38 @@ $middlewares = new MiddlewareRunner([
 ]);
 ```
 
+#### RequestBodyParserMiddleware
+
+Another built-in middleware is `RequestBodyParserMiddleware` which takes a fully buffered request body 
+(generally from [`RequestBodyBufferMiddleware`](#requestbodybuffermiddleware)), and parses the forms and uploaded files from the request body.
+
+Parsed submitted forms will be available from `$request->getParsedBody()` as array. For example the following submitted body:
+
+`bar[]=beer&bar[]=wine`
+
+Results in the following parsed body:
+
+```php
+$parsedBody = [
+    'bar' => [
+        'beer',
+        'wine',
+    ],
+];
+```
+
+Usage:
+
+```php
+$middlewares = new MiddlewareRunner([
+    new RequestBodyParserMiddleware(),
```

Comment:
Should probably have added the `RequestBodyBufferMiddleware` before to document the requirement.